source/api/image/3.0/index.md

```diff
@@ -477,92 +477,126 @@ Servers should also support [CORS][cors-response] on Image Information responses
-  ],
+  "protocol": "http://iiif.io/api/image",
+  "width": 6000,
+  "height": 4000,
   "tiles": [
-    {"width" : 512, "scaleFactors" : [1,2,4,8,16]}
-  ],
-  "profile" : [ "http://iiif.io/api/image/{{ page.major }}/level2.json" ]
+    {"width": 150, "height": 100, "scaleFactors": [ 1, 2, 4, 8, 16 ] }
+  ]
 }
 ```
 
-### 5.3. Profile Description
+### 5.5. Profile
 
-In order to specify additional features that are supported for the image, a profile object may be added to the `profile` list. Objects in the `profile` list have the properties in the following table. The `@context`, `id` and `type` properties are _REQUIRED_ when the profile is dereferenced from a URI, but _SHOULD NOT_ be included in the image information response.
+The JSON response also contains four properties that describe the functionality of the service. The `profile` property is required.
 
-| Profile Property | Required? | Description |
-| ----------- | --------- | ----------- |
-| `@context`  | Optional  | The string "http://iiif.io/api/image/{{ page.major }}/context.json". This should be included only if the profile's URI is dereferenced. |
-| `id`       | Optional  | The URI of the profile. |
-| `type`     | Optional  | The type of the object. If present, the value _MUST_ be the string "iiif:ImageProfile". |
-| `formats`   | Optional  | The set of image format parameter values available for the image.  If not specified then clients should assume only formats declared in the compliance level document.|
-| `maxArea`   | Optional  | The maximum area in pixels supported for this image. Requests for images sizes with width*height greater than this may not be supported. |
-| `maxHeight` | Optional  | The maximum height in pixels supported for this image. Requests for images sizes with height greater than this may not be supported. If `maxWidth` is specified and `maxHeight` is not, then clients should infer that `maxHeight = maxWidth`.  |
-| `maxWidth`  | Optional  | The maximum width in pixels supported for this image. Requests for images sizes with width greater than this may not be supported. _MUST_ be specified if `maxHeight` is specified. |
-| `qualities` | Optional  | The set of image quality parameter values available for the image.  If not specified then clients should assume only qualities declared in the compliance level document.|
-| `supports`  | Optional  | The set of features supported for the image.  If not specified then clients should assume only features declared in the compliance level document. |
+| Property   | Required? | Description |
+| ---------- | --------- | ----------- |
+| `profile` | Required | A string indicating the highest [compliance level][compliance-levels] which is fully supported by the service. The value _MUST_ be one of "level0", "level1", "level2". |
+| `extraQualities` | Optional | A list of strings that can be used in the quality parameter slot, in addition to the ones specified in the referenced profile. |
```

Comment:
should not introduce "slot" -- suggest s/used in the quality parameter slot/used as the quality parameter/ (and in following 2 lines)